---a/net/minecraft/network/login/server/S02PacketLoginSuccess.java
+++b/net/minecraft/network/login/server/S02PacketLoginSuccess.java
@@ -1,50 +1,44 @@
 package net.minecraft.network.login.server;
 
-import com.mojang.authlib.GameProfile;
 import java.io.IOException;
 import java.util.UUID;
+
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.login.INetHandlerLoginClient;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class S02PacketLoginSuccess implements Packet
-{
+import com.mojang.authlib.GameProfile;
+
+public class S02PacketLoginSuccess implements Packet {
+
     private GameProfile profile;
     private static final String __OBFID = "CL_00001375";
 
-    public S02PacketLoginSuccess() {}
+    public S02PacketLoginSuccess() {
+    }
 
-    public S02PacketLoginSuccess(GameProfile profileIn)
-    {
+    public S02PacketLoginSuccess(GameProfile profileIn) {
         this.profile = profileIn;
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         String s = buf.readStringFromBuffer(36);
         String s1 = buf.readStringFromBuffer(16);
         UUID uuid = UUID.fromString(s);
         this.profile = new GameProfile(uuid, s1);
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         UUID uuid = this.profile.getId();
         buf.writeString(uuid == null ? "" : uuid.toString());
         buf.writeString(this.profile.getName());
     }
 
-    public void processPacket(INetHandlerLoginClient handler)
-    {
+    public void processPacket(INetHandlerLoginClient handler) {
         handler.handleLoginSuccess(this);
     }
 
-    @SideOnly(Side.CLIENT)
-    public GameProfile getProfile()
-    {
-        return this.profile;
+    public void processPacket(INetHandler handler) {
+        this.processPacket((INetHandlerLoginClient) handler);
     }
-
-    public void processPacket(INetHandler handler)
-    {
-        this.processPacket((INetHandlerLoginClient)handler);
-    }
-}
+}
